integration_tests/integration/bookedtoday/unexpectedArrivals/searchForExistingRecord.spec.ts

```diff
@@ -16,23 +16,20 @@ context('Unexpected arrivals - Search for existing record spec', () => {
     SearchForExistingPage.goTo()
   })
 
-  it('Search with no details', () => {
+  it('Search without first name, last name or date of birth', () => {
     const searchPage = Page.verifyOnPage(SearchForExistingPage)
     searchPage.search().click()
-    searchPage.hasError("You must search using either the prisoner's last name, prison number or PNC Number")
-  })
-  it('Enter first name only', () => {
-    const searchPage = Page.verifyOnPage(SearchForExistingPage)
-    searchPage.firstName().type('James')
-    searchPage.search().click()
-    searchPage.hasError('Enter a last name')
+    searchPage.hasError(
+      "You must search using either the prisoner's last name and date of birth, prison number or PNC Number"
+    )
   })
 
-  it('Enter partial date only', () => {
+  it('Enter partial date of birth', () => {
```

Comment:
don't think you need separate tests for different validation rules - we already test these with jest

Reply:
see comment below